setup.py

```diff
@@ -1,9 +1,20 @@
 import setuptools
 
 
+with open("README.md", "r", encoding="utf-8") as desc:
+    long_description = desc.read()
+
+
 setuptools.setup(
     name="xtractmime",
-    version="0.0.0",
+    version="1.0.0",
+    license="BSD",
+    description="""Implementation of the MIME Sniffing standard 
+                   (https://mimesniff.spec.whatwg.org/)""",
```

Comment:
This will introduce unintended extra spaces. Use this instead:

```suggestion
    description=(
        "Implementation of the MIME Sniffing standard  (https://mimesniff.spec.whatwg.org/)"
    ),
```